.github/workflows/main.yaml

```diff
@@ -139,6 +139,14 @@ jobs:
           echo "MEDIATOR_URL=${MEDIATOR_URL}" >.env
           echo "IAS_PORTAL_URL=${IAS_PORTAL_URL}" >>.env
           echo "IAS_AGENT_INVITE_URL=${IAS_AGENT_INVITE_URL}" >>.env
+      
```

Comment:
This is my attempt at adding the push notification capability to IOS and adding the env variables. Obviously these shouldn't be hard coded.